Userland/Libraries/LibWebSocket/Impl/TCPWebSocketConnectionImpl.cpp

```diff
@@ -44,7 +44,7 @@ void TCPWebSocketConnectionImpl::connect(ConnectionInfo const& connection)
 
 bool TCPWebSocketConnectionImpl::send(ReadonlyBytes data)
 {
-    return m_socket->write(data);
+    return m_socket->write(data).is_error();
```

Comment:
Do you intend to work on `TCPWebSocketConnectionImpl::send` next? If not, please add something like:
`// TODO: Propagate errno code`

Reply:
Unrelated but same line: I think this should be inverted? Assuming that the return code means success.

Reply:
I did not; it has been added